webhook-integrations/Runscope/notes.md

```diff
@@ -0,0 +1,31 @@
+# Runscope Integration Investigation:-
+
+* [Runscope](https://runscope.com) is a company that provides solutions for API Performance Testing.
+
+* Runscope helps a company to conduct API Performance tests on an independent, hassle-free platform.
+
+* Runscope already has integrations with Slack & generates notifications in Slack for the following cases:
+
+  1. Test Passed
+
+  2. Test Failed
+
+
+* The Notification Model Image - https://github.com/MadElf1337/zulip-gci-submissions/blob/patch-13/webhook-integrations/Runscope/notification_model.PNG
+
+* The 'Test Passed' notification - https://github.com/MadElf1337/zulip-gci-submissions/blob/patch-13/webhook-integrations/Runscope/test_passed.PNG
+
+* The 'Test Failed' notification - https://github.com/MadElf1337/zulip-gci-submissions/blob/patch-13/webhook-integrations/Runscope/test_failed.PNG
```

Comment:
You should use relative markdown links for these images - [The 'Test Failed' notification](notification_model.PNG)